blog/why-i-joined-temporal-charles-zedlewski.md

```diff
@@ -62,47 +62,47 @@ Intermittent failures cost us in hard dollars with every order that gets dropped
 Engineers pay an equally large cost in productivity as they try to diagnose failures within a constellation of services written in various languages, on disparate infrastructure, each operating and logging differently from the other.
 
 Temporal upends all of this.
-Temporal is a new kind of platform, it’s not a database, cache, queue or a means to run your code.
+Temporal is a new kind of platform. It’s not a database, cache, queue, or a means to run your code.
 It’s a runtime for distributed applications that transparently manages their state at any scope and scale.
-By externalizing state to a generalized platform like Temporal, application teams get to outsource the majority of  cross-cutting concerns presented by cloud applications.
-In doing so applications become:
+By externalizing state to a generalized platform like Temporal, application teams get to outsource the majority of cross-cutting concerns presented by cloud applications.
+In doing so, applications become:
 
-- More reliable to run - Temporal applications fail to execute less often and when parts of the application do fail, they always recover to a consistent state
-- More productive to enhance - it’s not uncommon to see examples of engineers implementing an enhancement with 40-60% fewer lines of code than before as they outsource the most of the distributed application concerns to Temporal.
-- Easier to operate - Temporal helps users see how their production application executes its logic, consolidates errors and provides simple but powerful mechanisms to make fixes without interrupting the operation of the running application.
+- **More reliable to run:** Temporal applications fail to execute less often, and when parts of the application do fail, they always recover to a consistent state
+- **More productive to enhance:** It’s not uncommon to see examples of engineers implementing a feature in Temporal with 40-60% fewer lines of code than before, as they're outsourcing most of the distributed application concerns to Temporal.
+- **Easier to operate:** Temporal helps users see how their production application executes its logic, consolidates errors, and provides simple but powerful mechanisms to make fixes without interrupting the operation of the running application.
 
-At write time the Temporal SDK provides engineers a simple set of primitives native to their preferred language, the most central of which are Workflows.
+At write time, the Temporal SDK provides engineers a simple set of primitives native to their preferred language, the most central of which are Workflows.
```

Comment:
Perhaps change "At write time" to "During development"? (Like Loren, I found that this word choice caused a short stumble.)